contents/authors.md

```diff
@@ -15,6 +15,8 @@ authors:
       image: https://posthog.imgix.net/static/Mo-9fac6a2da6848cd8dfa6e1c339e15938.png?w=90,
       link_type: twitter,
       link_url: https://twitter.com/shehuphd
+      link_type: linkedin,
+      link_url: https://linkedin.com/in/shehuphd
```

Comment:
This is not possible unfortunately @shehuphd. These author entries are mappings where keys can't be duplicated. If we want to have multiple links per author, that's a small task for @smallbrownbike probably. :)

Reply:
Interesting - saw @joethreepwood added his LinkedIn URL and added mine for completeness. +1 on enabling multiple links.